.github/workflows/dev-build.yml

```diff
@@ -165,19 +173,15 @@ jobs:
           cargo build --bin mm2 --release
 
       - name: Compress build output
-        env:
-          AVAILABLE: ${{ secrets.FILE_SERVER_KEY }}
-        if: ${{ env.AVAILABLE != '' }}
+        if: env.FILE_SERVER_AVAILABLE
```

Comment:
Did you test this? The last time I tried it wasn't working or supported directly like this.

Reply:
Nope, I thought this should be intuitive, but truly nothing is intuitive about github actions :cry:.
The case you want to make sure of here is that when this is run from another repo it shouldn't fail here?

Reply:
https://github.com/mariocynicys/adex/actions/runs/8433919008/job/23096000988#step:12:28
Yup! you are right!

Reply:
In that case we have to revert those